### PR TITLE
Add utilities to setup roles and authorizations

### PR DESCRIPTION
**Pull request by petrus-v**

### CODE REVIEW

File: anyblok_pyramid/bloks/authorization/authorization.py [hunk 1]
```diff
@@ -189,3 +190,478 @@ def check_validity(self):
...
+                role, model, authz
+            )
+        Pyramid.Authorization.query().filter_by(role=role).filter(
+            ~Pyramid.Authorization.model.in_(config.keys())
+        ).delete(synchronize_session="fetch")
+        return role
+
+
+@Declarations.register(Pyramid.Authorization)
+class Configuration:
```
Comment (petrus-v) [inline anyblok_pyramid/bloks/authorization/authorization.py:279]: @jssuzanne I'm not sure `Configuration` is a good choice as far there are already some AnyBlok and Pyramid configuration ? Do you have any preferences ?

* Model.Pyramid.Authorization.**Configuration** (like it is right now I'm writing those line at commit cf304f38b25431d28233c4954f57613eea43f889)
* Model.Pyramid.Authorization.**Configurator** 
* Model.Pyramid.Authorization.**Setter**
* Model.Pyramid.Authorization.**Initializer**
* Model.Pyramid.Authorization.**Utils**
...

Comment (jssuzanne) [reply anyblok_pyramid/bloks/authorization/authorization.py:279]: I think a model is too much juste for some dict json

```
from anyblok_furetui import ACCESS_CRUD, ACCES_READ
from anyblok_furetui.common import ACCESS_CRUD, ACCES_READ
```

**Review by jssuzanne — CHANGES_REQUESTED** [review]

File: anyblok_pyramid/bloks/authorization/authorization.py [hunk 2]
```diff
@@ -189,3 +191,105 @@ def check_validity(self):
...
+                    modified = modified | {"perms"}
+                authz.flag_modified(*(jsonfields & modified))
+        return authz
+
+
+@Declarations.register(Pyramid)
+class Role:
+
+    @classmethod
+    def ensure_role_exists(
```
Comment (petrus-v) [inline anyblok_pyramid/bloks/authorization/authorization.py:241]: this name is a bit redundant, could be ensure_exists ! any thought ?

File: anyblok_pyramid/bloks/authorization/authorization.py [hunk 3]
```diff
@@ -189,3 +191,105 @@ def check_validity(self):
             raise AuthorizationValidationException(
                 "Primary keys without model to apply in the authorization "
                 "(%s)" % self)
+
+    @classmethod
+    def ensure_authorizations_exists(
```
Comment (petrus-v) [inline anyblok_pyramid/bloks/authorization/authorization.py:196]: probably ensure_authorization_exists would be better or ensure_exists as mentioned bellow ?

File: anyblok_pyramid/bloks/authorization/authorization.py (hunk 1 above)
Comment (petrus-v) [reply anyblok_pyramid/bloks/authorization/authorization.py]: I've made things shorter and exposed it as suggested

File: anyblok_pyramid/bloks/authorization/authorization.py (hunk 2 above)
Comment (petrus-v) [reply anyblok_pyramid/bloks/authorization/authorization.py]: done

File: anyblok_pyramid/bloks/authorization/authorization.py (hunk 3 above)
Comment (petrus-v) [reply anyblok_pyramid/bloks/authorization/authorization.py]: done

**Comment by petrus-v** [comment]
CI is red because using unreleased anyblok feature (`flag_modified`), waiting new AnyBlok release

**Review by jssuzanne — CHANGES_REQUESTED** [review]

**Comment by jssuzanne** [comment]
Thanks for all

**Pull request merged**